client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -492,6 +492,21 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         MOBILE_APP_LOCALE("mobileAppLocale", "en_US", false),
 
         /**
+         * This parameter represents the appium device.
```

Comment:
`appium` is not necessarily true here. Please update this comment and the ones for `MOBILE_PLATFORM_VERSION` and `MOBILE_DEVICE_TYPE`